cmd/istioctl/main.go

```diff
@@ -28,8 +32,22 @@ import (
 
 	"istio.io/manager/cmd"
 	"istio.io/manager/model"
+
+	// Note that to get this package I had to do
+	//     cd $GOPATH/src/k8s.io/client-go
+	//     git checkout v2.0.0
+	// because the default branch that `go get` uses couldn't find this package.
+	"k8s.io/client-go/pkg/util/yaml"
```

Comment:
No first person comments. I think this specific dependency needs to be encoded in bazel